tests/resources/test_branch.py

```diff
@@ -134,7 +135,8 @@ def test_branch_delete(session, collection, branch_path):
 def test_branch_update(session, collection, branch, branch_path):
     # Given
     branch.name = "NEW NAME"
-    session.set_response(branch.dump())
+    branch_data = BranchDataFactory(name=branch.name)
```

Comment:
probably can just go straight to `branch_data = BranchDataFactory(name="NEW NAME")` without using the `branch` fixture.